frontend/src/components/my_calendar/MyCalendar/MyCalendar.tsx

```diff
@@ -28,15 +33,36 @@ const MyCalendar = (props: MyCalendarProps) => {
     today,
     handlers: { handlePrevButtonClick, handleNextButtonClick },
   } = useCalendar();
-  const schedules = useFetchMySchedules(year, month);
+
   const { teamPlaces } = useTeamPlace();
+
+  const { firstDateOfCalendar, lastDateOfCalendar } =
+    getFirstLastDateOfCalendar(year, month);
+
+  const firstDateOfPreviousMonthCalendar = new Date(
```

Comment:
위 변수명의 형식대로 변수를 짓자 너무 길어지는 것 같아서..
```suggestion
  const firstDateOfPreviousCalendar 
```
이거 아니면
```suggestion
  const firstDateOfPrevCalendar
```
어때?

Reply:
이 요구사항을 반영하기 전에는 이전 달 / 다음 달의 일정을 삼항연산자로 그 자리에서 계산해서 요청했는데, 루루는 보통 따로 변수를 만들어서 대입하는 것을 좋아하는 것으로 알고 있어~ 리턴할 때에도 보통 리턴할 값을 변수로 정리한다음 리턴하는 걸 선호하는 것도 기억하고

일단 내가 봤을 때는 이렇게 내버려둬도 괜찮은 것 같고 따로 이전 달에 해당하는 변수 등을 `prevCalendarMonth` 과 같은 변수로 선언한 뒤
```tsx
const prevCalendarYear =  month === 0 ? year - 1 : year;
const prevCalendarMonth = month === 0 ? 11 : month - 1;

usePrefetchMySchedules(
  generateCalendarRangeByYearMonth(
    prevCalendarYear,
    prevCalendarMonth,
  ),
);
```
형태로 사용하는 것도 괜찮아 보이고~
어떤 방식을 좋아해?

Reply:
나는 아래꺼가 더 좋아보이네! 패치 안에 함수 안에 삼항연산자가 있으니까 가독성이 너무 떨어지는 것 같거든!